public/translations/cy.json

```diff
@@ -20,6 +20,7 @@
       "placeholder": "Rhowch enw eitem..."
     },
     "container": {
+      "subscription": "Gwasanaeth tanysgrifio – £{cost}",
```

Comment:
Does this need to be `{{cost}}` instead of `{cost}`?

Reply:
Yes it does you beautiful man